sympy/utilities/lambdify.py

```diff
@@ -21,21 +21,27 @@
 # by simple variable maps, like I => 1j
 # These are separate from the names above because the above names are modified
 # throughout this file, whereas these should remain unmodified.
+CMATH_DEFAULT = {}
 MATH_DEFAULT = {}
 MPMATH_DEFAULT = {}
 NUMPY_DEFAULT = {"I": 1j}
 SYMPY_DEFAULT = {}
 
 # Mappings between sympy and other modules function names.
+
+CMATH_TRANSLATIONS = {
+    "Abs": "abs",
+}
+
 MATH_TRANSLATIONS = {
-    "Abs": "fabs",
+    "Abs": "abs",
```

Comment:
I guess it is safe to do floating point division per default in lambdify. If one wants to do integer division he needs to be explicit by using sp.floor. WDYT?